build-es.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+#
+# create a docker image with ElasticSearch + our data by :
+#
+#   - running a fresh ES setup
+#   - ingest data to ES via code-du-travail-data scripts
+#   - create a new docker image with the ES data folder embedded
+#
+
+TMP_ES_LOCAL_PORT=9222
+TMP_DATA_PATH="$PWD/docker/elasticsearch/data"  # this must be under elasticsearch Dockerfile-with-data context
+
+docker rm -f cdtn-es || true
+rm -rf $PWD/docker/elasticsearch/data || true
+
+# build and launch custom ES image with local mount
+docker build ./docker/elasticsearch -t cdtn-es
```

Comment:
does `-t cdtn-es` option should be before PATH see https://docs.docker.com/engine/reference/commandline/build/#usage